api/controllers/ExportController.js

```diff
@@ -113,40 +121,85 @@ async function exportMembers(groupId, req, res) {
   }))
 
   // send data as CSV response
-  output(res, results, [
+  output(results, [
     'name', 'contact_email', 'contact_phone', 'location', 'avatar_url', 'tagline', 'bio',
     { key: 'url', header: 'personal_url' },
     'twitter_url', 'facebook_url', 'linkedin_url',
     'skills', 'skills_to_learn',
-    'join_request_questions',
     'affiliations',
     'groups'
-  ])
+  ], email, group.get('name'), questions)
 }
 
 // toplevel output function for specific endpoints to complete with
-function output(res, data, columns) {
-  res.setHeader('Content-Type', 'text/csv')
-  res.setHeader('Content-Disposition', 'attachment; filename=\"' + 'download-' + Date.now() + '.csv\"')
+function output(data, columns, email, groupName, questions) {
+  // Add each question as a column in the results
+  const questionsArray = Object.values(questions)
+  questionsArray.forEach((question) => {
+    columns.push(`${question.text}`)
+  })
 
-  stringify(data, {
+  // Add rows for each user to match their answers with the added question colums
+  const transformedData = data.map((user) => {
+    const answers = user.join_request_answers
+    questionsArray.forEach((question) => {
+      if (!answers) {
+        user[`${question.text}`] = '-'
+
```

Comment:
extra blank line, really gotta get our linting setup for node

Reply:
Will adjust